### PR TITLE
Me/dpc 4195 schedule verify ao job

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4195

## 🛠 Changes

- Scheduled VerifyAOJob to run weekly, every Sunday at 1am.
- Added cron tab to Sidekiq dashboard.
- Fixed "curl not found" bug in the portal Docker image and a typo in sidekiq-dev.

## ℹ️ Context

We need to schedule the VerifyAOJob to run weekly to verify that our authorized officials don't have any sanctions against them.

## 🧪 Validation

Initially scheduled the job to run every minute just to prove it was working:
Screenshot 2024-08-08 at 9 48 47 AM

Reschedule weekly and verified the cron was accurate:
image

### CODE REVIEW

File: dpc-portal/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ RUN apk update && \
     apk add --no-cache shared-mime-info && \
     apk add --no-cache --virtual build-deps alpine-sdk npm tzdata && \
     apk add xz && \
-    apk add --no-cache gcompat
+    apk add --no-cache gcompat && \
+    apk add --no-cache curl
```
Comment (MEspositoE14s) [inline dpc-portal/Dockerfile:15]: Not necessarily part of this ticket, but the sidekiq container was logging a "curl not found" error when it tried to generate a golden macaroon, and this was a quick fix.

File: dpc-portal/config/schedule.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+verify_ao_job:
+  cron: "* 1 * * 0"
+  class: "VerifyAoJob"
+  queue: default
```
Comment (jdettmannnava) [inline dpc-portal/config/schedule.yml:4]: Tested locally: needs to be
queue: portal

**Review by jdettmannnava — CHANGES_REQUESTED** [review]

File: dpc-portal/config/schedule.yml (hunk 2 above)
Comment (MEspositoE14s) [reply dpc-portal/config/schedule.yml]: @jdettmannnava was nice enough to point this out privately, but my schedule was wrong, too 😞.

**Review by jdettmannnava — APPROVED** [review]
LGTM

File: dpc-portal/config/schedule.yml (hunk 2 above)
Comment (jdettmannnava) [reply dpc-portal/config/schedule.yml]: I just wasn't sure!

**Review by lukey-luke — APPROVED** [review]
LGTM

**Pull request merged**